src/qml/export-edl/export-edl.js

```diff
@@ -4275,7 +4275,7 @@ process.nextTick = function (fun) {
     }
 };
 
-// v8 likes predictible objects
```

Comment:
export-edl.js is a generate file where this line came from a node.js npm module - more code I do not want to maintain.